### ISSUE TITLE
division by zero error while training Tacotron

### ISSUE THREAD

**Issue by YMaks** [Rayhane-mamah/Tacotron-2#278]
Hi. I'm trying to train both models sequentially on M-AILABS Speech Dataset with GPU GTX 1080 Ti. I've set max_mel_frames = 800 and left outputs_per_step = 1 (firstly, tried with outputs_per_step = 2, then decreased it as still didn't get OOM). But after writing summary at step 10000 I've got this error:
```
Running evaluation at step 10000 

0it [00:00, ?it/s]
Exiting due to exception: division by zero 

Traceback (most recent call last):
  File "/home/Tacotron-2/tacotron/train.py", line 257, in train
    linear_loss = sum(linear_losses) / len(linear_losses)
ZeroDivisionError: division by zero
```
Can anyone help me, please? I didn't get this error with default params and tacotron_batch_size = 16 but as was expected such batch size caused poor results.

**Comment by freds0** [comment]
I also had this problem. Check the version of the Tensorflow. Try 1.8.

**Comment by freds0** [comment]
The problem is in the evaluation. I could not solve it, but I overcame the problem by increasing the evaluation interval using the parameter in the training: --eval_interval 9999999

**Comment by ghost** [comment]
@freds0 Thanks for the tip, I've experienced the exact same issue. Do you know if missing out the eval will affect the training?

**Comment by freds0** [comment]
I do not think so, but I just gave up using this repository. I got good results with the Tacotron available in the NVIDIA repository.

Sorry for the delay, I did not see the response notification.

**Comment by m-toman** [comment]
Without looking at the code line now, usually it's just this https://github.com/Rayhane-mamah/Tacotron-2/issues/391

But if it didn't happen with a lower batch size probably not

**Comment by freds0** [comment]
@m-toman Now, I understood the problem. Thank you!
